giflib.go

```diff
@@ -136,6 +136,7 @@ func (d *gifDecoder) DecodeTo(f *Framebuffer) error {
 	if !ret {
 		return ErrDecodingFailed
 	}
+	f.duration = time.Duration(C.giflib_decoder_get_prev_frame_delay(d.decoder)) * 10 * time.Millisecond
```

Comment:
I think this is a bit clearer as:

```suggestion
	f.duration = time.Duration(C.giflib_decoder_get_prev_frame_delay(d.decoder) * 10 * time.Millisecond)
```

It might be worth a comment somewhere about `DelayTime`/`prev_frame_delay_time` being measured in whatever the name is for 0.01s 🤔